scst-store/using_encryption_and_connection.md

```diff
@@ -1,6 +1,6 @@
 # Using Encryption and Connecting to Supply Chain Security Tools - Store
 
-You can enable an encrypted connection using one of the following methods, depending on your service environment:
+The connection to the Store requires TLS encryption. Use one of the following methods to setup the TLS connection, depending on your service environment:
```

Comment:
I'm a bit confused on how setting the service type will help setup the TLS connection.

Reply:
It's because there are different steps depending on the type of service. I'll try to reword it to be clearer

Reply:
Does this look better?